docs/reference/data_context_reference.rst

```diff
@@ -260,20 +260,16 @@ new directory or use this template:
 
     # Welcome to Great Expectations! Always know what to expect from your data.
     #
-    # Here you can define datasources, generators, integrations and more. This file
-    # is intended to be committed to your repo. For help with configuration please:
+    # Here you can define datasources, batch kwarg generators, integrations and
+    # more. This file is intended to be committed to your repo. For help with
+    # configuration please:
     #   - Read our docs: https://docs.greatexpectations.io/en/latest/reference/data_context_reference.html#configuration
     #   - Join our slack channel: http://greatexpectations.io/slack
-    #
-    # NOTE: GE uses the names of configured `datasources` and `generators` to manage
-    # how `expectations` and other artifacts are stored in the `expectations/` and
-    # `datasources/` folders. If you need to rename an existing `datasource` or
-    # `generator`, be sure to also update the relevant directory names.
 
     config_version: 1
 
     # Datasources tell Great Expectations where your data lives and how to get it.
-    # You can use the CLI command ``great_expectations datasource new`` to help you
+    # You can use the CLI command `great_expectations add-datasource` to help you
```

Comment:
eyeball check...is the *new* syntax add-datasource?

Reply:
Lol, you have both passed my round robin test.